pkg/syncer/volume_health.go

```diff
@@ -110,9 +114,19 @@ func csiGetVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 			// updated every 5 mins, wait for an hour or so before taking any
 			// corrective actions. This is an acceptable level of eventual
 			// consistency.
-			updateVolumeHealthStatus(ctx, k8sclient, pvc, common.VolHealthStatusInaccessible)
+			volHealthStatusAnn = common.VolHealthStatusInaccessible
```

Comment:
Why we set volHealthStatusAnn to "inaccessible" at line 117?

Reply:
@lipingxue This was being passed already to the function on line 118. I've just assigned it to a variable so that i can increment the count on line 124. There's no change in logic here.

Reply:
Got it.